src/components /Pokemon.js

```diff
@@ -0,0 +1,46 @@
+import React, {useEffect, useState} from 'react';
+import axios from "axios";
+import './Pokemon.css';
+
+function Pokemon({endpoint}) {
+
+    const [pokemonData, setPokemonData] = useState({});
+    useEffect(() => {
+
+        async function getPokemons() {
+            try {
+                const {data} = await axios.get(endpoint);
+                console.log(data)
+                setPokemonData(data);
+            } catch (e) {
+                console.log(e);
+            }
+        }
+
+        getPokemons()
+    }, [endpoint])
+
```

Comment:
Als je hier
```suggestion
    if (!pokemonData) return null;
```
doet, dan kun je de check die hieronder staat weghalen en ben je wat nesting kwijt